bugbot/rules/not_landed.py

```diff
@@ -66,6 +66,21 @@ def handle_bug(self, bug, data):
 
         return bug
 
+    def update_revision_status(self, rev_id, status_type, value):
+        try:
+            self.phab.request(
+                "differential.revision.edit",
+                objectIdentifier=rev_id,
+                transactions=[{"type": status_type, "value": value}],
+            )
+            print(
+                f"Successfully updated revision {rev_id} with {status_type} to {value}"
+            )
+        except Exception as e:
+            print(
+                f"Failed to update revision {rev_id} with {status_type} to {value}: {e}"
+            )
```

Comment:
This is a big enough change to the process that it might be controversial. We should just start by not needinfoing for patches that have been backed out, and later on we could consider automatically marking them as changes-planned.

Reply:
Removed here: 0c4e10e272f7b16fb6f9b45f543e9ae7ba7629c0